packages/l10n/src/index.ts

```diff
@@ -18,3 +18,8 @@ export {
   withTimeZones,
   timeZonesShape,
 } from './time-zone-information';
+
+export {
+  transformLocalizedFieldToLocalizedString,
+  injectTransformedLocalizedFields,
```

Comment:
hey. I like that name!
let's go for it.

regarding your question, given the status quo it made sense to expose `transformLocalizedFieldToLocalizedString` as well since it is used all over MC. Now that I think of it, maybe we can change that and not expose it.

It probably makes more sense to not expose this at all actually.

maybe you can chime in on this, @tdeekens?

---

the big benefit of not exposing `transformLocalizedFieldToLocalizedString` is encourage "same pattern" on transformations.
we can expose this later again, if we want to update the merge policy in MC-FE's apollo cache to store the transformed field.

Reply:
Okay, here is what we do. We remove this now, because I like the idea to encourage just one tool.
I will update the documentation as well to remove `transformLocalizedFieldToLocalizedString`